data/repos.yml

```
@@ -338,6 +338,13 @@
   sentry_url: false
   private_repo: true
 
+- repo_name: govuk-graphql
+  team: "#govuk-publishing-platform"
+  alerts_team: "#govuk-publishing-platform"
```

Could this be updated to use our automated alerts channel
```suggestion
  alerts_team: "#govuk-publishing-platform-system-alerts"
```

---

Yep, Aga caught it too - fixed. Thanks!